### PR TITLE
lowers instability of antenna and removes it's access to command and sec channels

### PR DESCRIPTION
**Pull request by Marmio64**
### Intent of your Pull Request
lowers instability of antenna and removes it's access to command and sec channels

### Why is this change good for the game?
invalidates the privacy of sec and command comms way too easily, this fixes that

### What should players be aware of when it comes to the changes your PR is implementing?
antenna doesn't let people access sec and command channels now
also it's instability is 10 instead of 15 now since it's most powerful feature of having sec and command comms was removed.

### What general grouping does this PR fall under? 
mutation/genetics changes

### Are there any aspects of the PR that you would like us not to mention on the Wiki?
no?

:cl:  
rscdel: Removes antenna's access to the sec and command channels  
tweak: antenna mutation instability lowered to 10
/:cl:

### CODE REVIEW

**Review by SomeguyManperson — APPROVED** [review]

**Comment by athena413** [comment]
would rather just see it be for common only, since departments like engineering and others should still keep their privacy

**Comment by MegaEmpirical** [comment]
I'd support just common but this is nice too.

**Comment by Cartlord** [comment]
But if it only gives common access, what is the POINT of the mutation? What would one gain from it?

**Comment by Marmio64** [comment]
> But if it only gives common access, what is the POINT of the mutation? What would one gain from it?

well you could use it with earmuffs to have flashbang protection while keeping a radio

**Comment by Tipy1802** [comment]
> But if it only gives common access, what is the POINT of the mutation? What would one gain from it?

Crying for help over comms even when having your headset removed.
Thats how it used to work a few months ago before someone gave it all comms.
I am happy as long as it doesnt have sec or command comms, good pr

**Comment by athena413** [comment]
> 
> 
> But if it only gives common access, what is the POINT of the mutation? What would one gain from it?

it also makes your comms be EMP immune

**Comment by Cartlord** [comment]
Consider how ludicrously situational this is. Who's going to bother? And if its only real use is for if you get your headset stolen - which nobody but an antag or griefer would do - wouldn't the whole use of this count as powergaming?

**Comment by SomeguyManperson** [comment]
actually yeah it should go back to just common since geneticists tend to be shitters when they have access to things they shouldn't

**Comment by Marmio64** [comment]
> Consider how ludicrously situational this is. Who's going to bother? And if its only real use is for if you get your headset stolen - which nobody but an antag or griefer would do - wouldn't the whole use of this count as powergaming?

whole use right now already is just powergaming since you get sec and command access

File: code/game/objects/items/devices/radio/encryptionkey.dm [hunk 1]
```diff
@@ -130,3 +130,6 @@
 
 /obj/item/encryptionkey/secbot
 	channels = list(RADIO_CHANNEL_AI_PRIVATE = 1, RADIO_CHANNEL_SECURITY = 1)
+
+/obj/item/encryptionkey/antenna //for antenna mutation and it's unique demands. hopefully it wont be pulled out somehow.
```
Comment (Hopekz) [inline code/game/objects/items/devices/radio/encryptionkey.dm]:
```suggestion
/obj/item/encryptionkey/antenna //for antenna mutation and it's unique demands. hopefully, it won't be pulled out somehow.
```

**Comment by UselessTheremin** [comment]
hopek "spellcheck" strikes again

File: code/game/objects/items/devices/radio/encryptionkey.dm (hunk 1 above)
Comment (Marmio64) [reply code/game/objects/items/devices/radio/encryptionkey.dm]: its a comment, nobody cares about grammar

Comment (Hopekz) [reply code/game/objects/items/devices/radio/encryptionkey.dm]: > its a comment, nobody cares about grammar

I just read it and thought, "who the hell doesn't know their won't spellings" enough to want to check who the author is, so whether you like it or not, people care, and it DOES make you look bad.

There isn't anything wrong with looking illiterate, so it's ultimately up to you. I'm not forcing you to take the suggestion.

**Pull request merged**